newsfragments/146.misc.rst

```diff
@@ -0,0 +1 @@
+Updated test suite to cope with Trio 0.25.0 and later defaulting ``strict_exception_groups`` to ``True``. Trio 0.25.0 is now required to run the tests, although `~trio-asyncio` itself still supports older versions.
```

Comment:
Note the failing docs build here - single backticks in ReST are a cross-reference, and there's no entity named "trio-asyncio". (Also, there's no need for the tilde; the purpose of that is to not print the portion of the xref before the final dot, but there are no dots here.) You can reference the module "trio_asyncio", or double the backticks for fixed-width font without a cross-reference, but I would just drop the backticks and let trio-asyncio be like a normal word; that's the style we try to use in the rest of the documentation.